lib/Parser.js

```diff
@@ -429,7 +429,10 @@ function parseFetch(text, literals, seqno) {
   // list is [KEY1, VAL1, KEY2, VAL2, .... KEYn, VALn]
   for (var i = 0, len = list.length, key, val; i < len; i += 2) {
     try {
-      key = list[i].toLowerCase();
+      key = list[i];
+      if (typeof (key.toLowerCase) == 'function') {
+        key = key.toLowerCase();
+      }
```

Comment:
minor issue, sometimes key is an array, and therefore can't `toLowerCase()` on it.